packages/components/src/internal/components/domainproperties/PropDescType.ts

```diff
@@ -361,6 +363,7 @@ export const PROP_DESC_TYPES = List([
     VISIT_ID_TYPE,
     UNIQUE_ID_TYPE,
     TEXT_CHOICE_TYPE,
+    AUTOINT_TYPE,
```

Comment:
I pulled down the change locally to give it a try. The repro mentioned in the issue works as expected now, but I did see "Auto Increment" as an option in the data type dropdown for a new field in the designer. I don't think we want that. Would this work better if `AUTOINT_TYPE` was added to `READONLY_DESC_TYPES` below instead of here?
<img width="905" alt="Screenshot 2024-06-04 at 10 37 36 AM" src="https://github.com/LabKey/labkey-ui-components/assets/6411206/b20a4d1e-9cee-4464-87c3-30d74d9b71ec">

Reply:
Ah, good catch, thank you!